action/action.go

```diff
@@ -169,3 +189,13 @@ func execCommand(args ...string) string {
 	}
 	return string(stdout.Bytes())
 }
+
+// importFromFile reads the inputted file from filePath and returns
+// it b64encoded.
+func importFromFile(filePath string) string {
+	scanfile, err := ioutil.ReadFile(filePath)
+	if err != nil {
```

Comment:
Wondering here if rather that erroring if the file doesn't exist, we should instead print a message and let the warning be raised in line https://github.com/hashicorp/actions-generate-metadata/pull/15/files#diff-12d49a3ddd421707244d51ae2fa9073c2155999f60d594e926491b72637402d6R131. There may be cases when repos don't have these files for some reason (e.g only using CRT for build/internal releases therefore no release metadata needed e.g [Enos](https://github.com/hashicorp/enos/tree/main/.release))